config.json

```diff
@@ -14,6 +14,11 @@
     "point-mutations"
   ],
   "exercises": [
+	{
+		"difficult": 1,
+		"slug": "hello-world",
+		"topics": []
```

Comment:
Couple things here:

Make sure you turn off tabs and format this just like the rest of the file using spaces.

Spelling of `difficulty`

Also, please add "strings" to the `topics`. I know there aren't any for the other problems here, but we're supposed to be starting to add them :) You can look at https://github.com/exercism/xc/blob/master/config.json if you want to look at their formatting.